contrib/k8s/examples/mysql/mysqldb-database-only-binding.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: mysql-database-only-binding
```

Comment:
I would also suggest prefixing k8s resource names with something like "my-" so we'e making it more clear that there's no specific expectation that resource (instance and binding) names _match_ service names.

Reply:
done!